Source/Krypton Explorer/Form1.cs

```diff
@@ -396,5 +376,15 @@ private void kbtnRestoreTheme_Click(object sender, EventArgs e)
         }
         
         private void klblPrintDialog_LinkClicked(object sender, EventArgs e) => LaunchApplication(@"Krypton Print Dialog Example");
+
+        private void kbtnKryptonToolkitPackage_Click(object sender, EventArgs e)
+        {
+            Process.Start(@"https://www.nuget.org/profiles/Krypton_Suite");
+        }
+
+        private void kbtnViewLatestReleaseNotes_Click(object sender, EventArgs e)
```

Comment:
If this is the Latest, then should it not goto alpha ?
Or add an Alpha equivalent ?

Reply:
I can add a toggle in the code to switch between alpha & master changelogs?

Reply:
2 separate buttons would make it clearer